src/mame/drivers/cedar_magnet.cpp

```diff
@@ -1061,6 +1061,46 @@ ROM_START( mag_day )
 	ROM_LOAD( "adayinspace 31_3_87.img", 0x00000, 0xf0000, CRC(bc65302d) SHA1(6ace68a0b5f7a07a8f5c318c5359011074e7f2ec) )
 ROM_END
 
+// track 79 side 1 (file offset 0x0ee800) would not read, but it appears to be outside of the used data
+// this also has many scores stored on the disk at offset 0x01cc00, invalidating these does reset the
+// score table to 'EFO 100000' scores, but then it never writes new scores?
+ROM_START( mag_pdak )
+	BIOS_ROM
+
+	ROM_REGION( 0x100000, "flop:disk", ROMREGION_ERASE00 )
+	ROM_LOAD( "paris.dsk 31_3_87.img", 0x00000, 0xf0000, CRC(2c4ee9e1) SHA1(22c2b75c16aca95ecf2199451c1bd12dd3a3844c) )
+ROM_END
```

Comment:
If one track didn’t read, it should still be marked `BAD_DUMP`.

Reply:
it's outside the used area, game never reads it, typically we don't bother in such cases as it's leftover garbage that differs from disk to disk anyway.